src/core/meshcop/commissioner.hpp

```diff
@@ -78,6 +79,40 @@ class Commissioner
     ThreadError Stop(void);
 
     /**
+     * This method adds a Joiner entry.
+     *
+     * @param[in]  aExtAddress      A reference to the Joiner's extended address.
```

Comment:
I'd suggest we provide a constant for such a Wildcard Joiner Address and reference that here.  The reason to support of Wildcard is that it is much less typing for a user in the single Joiner case when no QR code reader is available. (PSKd = 6+ characters typed in, ExtAddress = 16 characters hex).  Can be a follow-on PR.